gradle/libs.versions.toml

```diff
@@ -3,42 +3,43 @@ javaVersion = "17"
 
 kotlinVersion = "1.9.25" # Kotlin and Dokka versions should have the same minor version to support the same kotlin version
 dokkaVersion = "1.9.20" # Kotlin and Dokka versions should have the same minor version to support the same kotlin version
-agpVersion = "8.6.0"
+agpVersion = "8.6.1"
 nexusPublishVersion = "2.0.0"
 ktlintVersion = "11.5.0"
 
 # be carefully with updating this plugin, it may upgrade kotlin version for the entire project
 # check the release notes and set the version similar with the project kotlin version
 # https://github.com/cashapp/paparazzi/releases
-paparazziVersion = "1.3.1"
+paparazziVersion = "1.3.4"
 
 javaAppCompatVersion = "1.7.0"
 javaConstraintLayoutVersion = "2.1.4"
 javaCoreKtxVersion = "1.13.1"
-javaKtorVersion = "2.2.4"
-javaCoroutinesVersion = "1.8.1"
-javaDokkaVersion = "1.8.20"
-javaLifecycleProcessVersion = "2.8.4"
-javaLintVersion = "31.6.0"
+javaKtorVersion = "3.0.0"
+javaCoroutinesVersion = "1.9.0"
+javaDokkaVersion = "1.9.20"
+javaLifecycleProcessVersion = "2.8.6"
+javaLintVersion = "31.7.1"
 javaMaterialVersion = "1.12.0"
 javaPreferenceVersion = "1.2.1"
-javaRxAndroid3Version = "3.0.0"
-javaRxJava3Version = "3.1.8"
-javaNavVersion = "2.8.0"
-dataGsonVersion = "2.10.1"
-firebaseBomVersion = "33.2.0"
-mediaAudioswitch = "1.1.8"
-mediaPicassoVersion = "2.8"
-mediaLottieVersion = "5.2.0"
+javaRxAndroid3Version = "3.0.2"
+javaRxJava3Version = "3.1.9"
+javaNavVersion = "2.8.3"
+dataGsonVersion = "2.11.0"
+firebaseBomVersion = "33.4.0"
+mediaAudioswitch = "1.2.0"
+mediaPicassoVersion = "2.71828"
+mediaLottieVersion = "6.5.2"
+mediaExifInterfaceVersion = "1.3.7"
```

Comment:
I expected to see only updated versions. However, `mediaExifInterfaceVersion` is a new one, it seems. Is this correct?

Reply:
Yes, because previously, the `android.core-ktx` library transitively contained this dependency, but in newer versions, they dropped it.